srcpkgs/linux5.5/files/x86_64-dotconfig

```diff
@@ -294,8 +295,7 @@ CONFIG_HAVE_INTEL_TXT=y
 CONFIG_X86_64_SMP=y
 CONFIG_ARCH_SUPPORTS_UPROBES=y
 CONFIG_FIX_EARLYCON_MEM=y
-CONFIG_DYNAMIC_PHYSICAL_MASK=y
-CONFIG_PGTABLE_LEVELS=5
+CONFIG_PGTABLE_LEVELS=4
```

Comment:
I'm against this one. There are some newer CPUs that benefit from this.

Reply:
Not really. 5-level page table is only useful if you have more than 256TiB of RAM and on everything else it adds overhead.